src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -178,27 +63,48 @@
                   :number-of-lines 5}
       text]]))
 
+(def ^:private kind->props-fn {:bold   (constantly {:style {:font-weight :bold}})
+                               :italic (constantly {:style {:font-style  :italic}})
+                               :link   (fn [text {:keys [outgoing]}]
+                                         {:style    {:color                (if outgoing colors/white colors/blue)
+                                                     :text-decoration-line :underline}
+                                          :on-press #(re-frame/dispatch [:browser.ui/message-link-pressed text])})
+                               :tag    (fn [text {:keys [outgoing]}]
+                                         {:style    {:color                (if outgoing colors/white colors/blue)
+                                                     :text-decoration-line :underline}
+                                          :on-press #(re-frame/dispatch [:chat.ui/start-public-chat (subs text 1)])})})
+
+(defn- render-chunks [render-recipe message]
+  (map-indexed (fn [idx [text kind]]
+                 [react/text
+                  (merge {:key idx}
+                         (apply merge-with merge
+                                (keep (comp #(when % (% text message)) kind->props-fn) kind)))
```

Comment:
`(comp #(when % (% text message)) kind->props-fn)` this part is a bit hard to read, can we extract it and give some name?

Reply:
I agree, will extract it to the separate function.